some-service/client.go

```diff
@@ -0,0 +1,70 @@
+// Package some_service
+package some_service
+
+import (
+	"GithubSearch/types/v1/api"
+	"GithubSearch/utils"
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/flannel-dev-lab/cyclops/v2/logger"
+	"net/http"
+	"time"
+)
+
+const hostname = "http://localhost"
+
+type SomeService interface {
+	ResultParser(ctx context.Context, searchResponse *api.SearchResponse) error
+}
+
+type Client struct {
+	httpClient *http.Client
+}
```

Comment:
Put the addr in the client

```suggestion
type Client struct {
	addr string
	httpClient *http.Client
}
```